spec/spec.md

```diff
@@ -23,7 +23,7 @@ The below are the most stable versions of the lanuguage specification, which are
 
 | Version | Release Date | Description |
 | ------- | ------------ | ----------- | 
-| <a target="_blank" href="/spec/lang/2022R1/">2022R1</a> | 2021-06-02 | First release of 2022. This is the basis for Ballerina 2201.0.0 (Swan Lake). |
+| <a target="_blank" href="/spec/lang/2022R1/">2022R1</a> | 2022-02-01 | First release of 2022. This is the basis for Ballerina 2201.0.0 (Swan Lake). |
```

Comment:
Or
```suggestion
| <a target="_blank" href="/spec/lang/2022R1/">2022R1</a> | 2022-01-31 | First release of 2022. This is the basis for Ballerina 2201.0.0 (Swan Lake). |
```

@hasithaa, @anupama-pathirage, thoughts please?

Reply:
+1 for 01-31